### DISCUSSION TITLE
Rezize Compass button

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#600] [Q&A] [answered]
Is there an option to rezize (enlarge) the compass button? The button size option doesn't work for the compass.

**Comment by devemux86** [comment]
@DolomitesTom thanks for the suggestion.

It was also requested [here](https://github.com/devemux86/cruiser/discussions/405#discussioncomment-9559606) and button size setting will also adjust the compass in next Cruiser.

**Comment by devemux86** [comment]
Button size setting now also adjusts the compass size:

- https://github.com/devemux86/cruiser/discussions/610

**Comment by devemux86** [comment, marked as answer]
- Button size setting also adjusts the compass size in https://github.com/devemux86/cruiser/discussions/641
